.env.example

```diff
@@ -0,0 +1,5 @@
+STORAGES_BACKEND=storages.backends.file_system.FileSystemStorage
+STORAGES_PATH=.
+STORAGES_AWS_ACCESS_KEY_ID=AKIACN3948TVUEXAMPLE
+STORAGES_AWS_SECRET_ACCESS_KEY=93hgocns02yt690gjeof95hgkgREXAMPLESECRET
+STORAGES_BUCKET_NAME=your-bucket-name
```

Comment:
what about adding Google Cloud examples also? We have it in README but if we have S3 let's add all of it here.